src/Components/Dashboard/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from 'react-router-dom';
+import { useTheme } from '../../context/ThemeContext';
+import light_mode_black_24dp from '../../database/assets/light_mode_white_24dp.svg';
+import dark_mode_black_24dp from '../../database/assets/dark_mode_black_24dp.svg';
+export const Header = () => {
+	const { theme, setTheme } = useTheme();
+
+	return (
+		<nav className=' md:mx-8 flex justify-between items-center px-6 py-4 border-b '>
+			<Link to='/dashboard' className=''>
+				<div className='text-purple-700 dark:text-purple-600 italic font-black text-lg'>
```

Comment:
Using `NavLink` is recommeded, in place of `Link` in Navbars.

Reply:
Done